examples/deploy_ecs/terraform/README.md

```diff
@@ -0,0 +1,29 @@
+# Dagster AWS ECS Deployment Example - Terraform
+
+This example demonstrates how to deploy Dagster on AWS ECS using Terraform.
+
+Two Terraform modules are provided:
+- [dagster-deployment](./dagster-deployment): responsible for deploying the Dagster Daemon and Dagster Webserver as ECS services.
+- [dagster-code-location](./dagster-code-location): responsible for deploying code locations as ECS services.
+
+The example expects the following resources to be used in the deployment:
+- a VPC with public and private subnets
+- a configured ECS cluster
+- a Postgres database for storing the Dagster event log
+
+During the deployment, the following resources are created:
+- IAM roles and policies for ECS tasks (optional, can be provided instead)
+- CloudWatch log group for the ECS tasks (optional, can be provided instead)
+- ECS services for the Dagster Daemon and Dagster Webserver
+- ECS services for code locations
+- AWS Application Load Balancer for the Dagster Webserver (optional, can be provided instead)
+
+By default, the `dagster-deployment` automatically creates basic IAM roles and policies for the ECS tasks, and deploys an AWS Application Load Balancer for the Dagster Webserver. **This will make the Dagster Webserver accessible from the public internet**, which is useful for a quickstart, testing or development purposes. Make sure to use public subnets for `webserver_subnet_ids` if you want to access the webserver from the internet.
```

Comment:
```suggestion
By default, the `dagster-deployment` automatically creates basic IAM roles and policies for the ECS tasks, and deploys an AWS Application Load Balancer for the Dagster Webserver. **This will make the Dagster Webserver accessible from the public internet**, which is useful for a quickstart, testing or development purposes. Make sure to use public subnets for `webserver_subnet_ids` if you want to access the webserver from the internet without a load balancer.
```

then other considerations -- are you in a public or private subnet? https without a load balancer? does the webserver need egress?